teams/migration/membership.json

```diff
@@ -38,11 +38,13 @@
         "3AceShowHand",
         "asddongmen",
         "ben1009",
+        "D3Hunter",
         "gozssky",
         "iamxy",
         "okJiang",
         "sdojjy",
-        "tiancaiamao"
+        "tiancaiamao",
+        "WizardXiao"
```

Comment:
Seems the name is sorted by ascii order, but I found `hi-rustin` is out of order😂